src/js/routes/jobs.js

```diff
@@ -3,7 +3,7 @@ import React from "react";
 /* eslint-enable no-unused-vars */
 import { IndexRoute, Redirect, Route } from "react-router";
 
-import JobsTabContainer from "#PLUGINS/jobs/src/js/JobsTabContainer";
```

Comment:
This is the line we would need to change back to not have it active, right?

Reply:
yes - and restore all the old files (I'd go with `#SRC/js/…` in this case)